transform/transformations/common.py

```diff
@@ -22,87 +24,35 @@
     UNIFIED_CATEGORIES,
     DOWNLOADS,
     VIEWS,
+    SUBJECT,
     LANGUAGE,
     TAG_LIST_TG,
     TAG_LIST,
     KEYWORDS_TG,
     KEYWORDS,
     AUTHOR_NAMES_TG,
+    PROJECTS,
+    INSTANCE,
+    CONTEXT,
+    PID,
 )
-
-# Mappings - values are mapped to the keys
-# Access rights mapping
-_OPEN_ACCESS = "Open access"
-RESTRICTED = "Restricted"
-ORDER_REQUIRED = "Ordered required"
-LOGIN_REQUIRED = "Login required"
-LOGIN_REQUIRED_ON = (
-    "Login required on EOSC Pillar, open access on the original resource page"
+from transform.schemas.mappings import (
+    _OPEN_ACCESS,
```

Comment:
If we import `_OPEN_ACCESS` from other modules it should not be prefixed with `_`